.eslintrc.js

```diff
@@ -11,5 +11,38 @@ module.exports = {
     'lines-around-comment': 'off',
 
     '@shopify/jsx-no-complex-expressions': 'off',
+
+    // this is the same as in @shopify/eslint-plugins
+    // except no typeParameter
```

Comment:
During our pairing session, we felt like it was making things more verbose as we always have to start type params with a `T`. But we can revert.

Reply:
Sure! This is triggering whenever we use non-prefixed generic arguments. So, take the `on` function on [packages/ui-extensions-server-kit/src/ExtensionServerClient/types.ts](https://github.com/Shopify/shopify-cli-extensions/pull/178/files#diff-fd2a2f32e2a53ea1019426b3a217297af66c78d7adc1b0bc3e595b43e3b384daR77). If you remove the `T` prefix, you'll get:

![Screen Shot 2021-11-12 at 14 57 15](https://user-images.githubusercontent.com/10423749/141533709-1971a7a6-b826-4978-acec-da5b04c6c30f.png)

Reply:
Albeit more verbose, I feel it's better to have it clearly stating what it means. Currently, the arguments are defined as `TEvent`, `TKey` and so on. My argument for this is that this should be used by other developers if they are using the kit and it's confusing to have single char arguments (especially `T` since it's so generic and has no direct meaning within this context... `E` could make more sense?). I feel this is the same as a function arguments just related to types.